src/content/docs/tutorial-errors/route-teams.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Assign errors to teams and update error status
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+import apmErrorsAssign from 'images/apm_screenshot-crop_errors-triage.webp'
+
+import apmErrorsStatus from 'images/apm_screenshot-crop_errors-status.webp'
+
+import apmErrorsAssignment from 'images/apm_screenshot-crop_errors-assignment.webp'
+
+
+Now that you understand the full scope of your outage and the error group, you can assign the error and update its status. When you assign errors within New Relic, you can transfer all the information you've gathered to the code owners. Managing your errors inbox eliminates information silos across multiple teams, which expedites the resolution process. 
```

Comment:
Good intro

Reply:
Agree, really nice intro. The last sentence though feels jargon-y and a bit boring:

> Managing your errors inbox eliminates information silos across multiple teams, which expedites the resolution process. 

What's a version of this that makes me excited to set this up as the first thing i do in my workday;/